packages/core/src/hooks.ts

```diff
@@ -51,7 +51,7 @@ export function initializeConnector<T extends Connector>(
 }
 
 function computeIsActive({ chainId, accounts, activating }: Web3ReactState) {
```

Comment:
Can you expand the test to cover a case with `accounts = []` and `chainId = undefined`?